fetch_data/cloud_api_data/get_cloud_api_data.py

```diff
@@ -0,0 +1,55 @@
+"""fetch list of wikibase cloud instances from api and update local database"""
+
+from dataclasses import dataclass
+from logger import logger
+
+from fetch_data.utils import fetch_api_data
+
+
+@dataclass
+class WikibaseCloudInstance:
+    """cloud instance as fetched from the wikibase cloud dashboard api"""
+
+    id: int
+    description: str | None
+    domain: str
+    domain_decoded: str
```

Comment:
Maybe it was obvious to you but wouldn't necessarily have been obvious to me:

This is a unicode string and the plain `domain` is PUNYcode. c.f.
`"domain":"xn--brgerspitalzinshaus-59b.wikibase.cloud","domain_decoded":"b\u00fcrgerspitalzinshaus.wikibase.cloud"`

probably a bit edgecasey but obviously something that bit us :)